nixos/modules/services/networking/pixiecore.nix

```diff
@@ -0,0 +1,112 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pixiecore;
+in
+{
+  options = {
+    services.pixiecore = {
+      enable = mkEnableOption "Pixiecore";
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
(chanelling IRC discussion) Default here should be false. Except SSH, NixOS doesn't openFirewall by default, even if the module is enabled.